quantmsio/commands/convert/quantms.py

```
@@ -0,0 +1,549 @@
+        )
+
+        logger.info("Processing complete!")
+
+        # Log some basic statistics
+        try:
+            metadata_count = len(indexer.get_metadata())
+            protein_count = indexer.get_protein_count()
+            protein_details_count = indexer.get_protein_details_count()
+            psm_count = indexer.get_psm_count()
+
+            logger.info(f"Successfully processed:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)